tests/ruff.toml

```diff
@@ -1,4 +1,4 @@
-extend-ignore =  ["E501", "G004", "PT004", "PT005", "RET504", "SIM114", "TRY003", "TRY400", "TRY401", "RUF012", "RUF100"]
+extend-ignore =  ["E501", "G004", "PT004", "PT005", "RET504", "SIM114", "TRY003", "TRY400", "TRY401", "RUF012", "RUF100", "ISC001"]
 extend-exclude = ["imaginairy/vendored", "downloads", "other"]
```

Comment:
When I ran the ruff formatter it recommended I add this exception as it would cause conflicts with some other rule.